PyInstaller/utils/win32/winutils.py

```diff
@@ -114,7 +106,12 @@ def import_pywin32_module(module_name):
 
             # If isolated to a venv, the preferred site.getsitepackages()
             # function is unreliable. Fallback to searching "sys.path" instead.
-            if compat.is_venv:
+            if _is_venv is None:
+                # prevents circular import
```

Comment:
Is this related to the import-statement? Then maybe it should read something like
"Don't import globally to prevents circular import"
and be put to all these nested import statements.